vsintegration/tests/FSharp.Editor.Tests/Hints/HintTestFramework.fs

```diff
@@ -15,6 +18,13 @@ module HintTestFramework =
     type TestHint =
         { Content: string; Location: int * int }
 
+    // like: C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.0\ref\net7.0\mscorlib.dll
+    let locateMscorlib() =
```

Comment:
This doesn't look very cross platform compatible. (Not that I know what to do about it.)

Reply:
Although maybe fcs is slightly different.

Reply:
@0101 @KevinRansom okay I think I found a better project option to achieve the same effect and it also align with some other test suites we have. Now all is green :)